packages/app/jest.config.js

```diff
@@ -29,7 +26,22 @@ module.exports = {
   roots: ["<rootDir>/src"],
   testResultsProcessor: "./jest.processor.js",
   collectCoverage: false,
-  collectCoverageFrom: ["src/**/*.{js,ts,tsx}"],
+  collectCoverageFrom: [
+    "src/**/*.{js,ts,tsx}",
+    "!src/__deprecated__/**",
```

Comment:
Maybe we should add `testing-support` folder as well?

Reply:
https://github.com/mudita/mudita-center/pull/776/commits/85a7b9c858cd28224986a43016aa8df47945f3a2